### PR TITLE
Create scaffolding for opentelemetry-kube-stack chart

### LINKED ISSUE #1075: [opentelemetry-kube-stack] Create hierarchy for the chart
### original issue
#562

### context
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1058

### description
Create the hierarchy needed for the chart to exist in the repository

### PR DESCRIPTION
**Pull request by jaronoff97**
This PR creates the scaffolding for the opentelemetry-kube-stack chart. It does nothing outside of setup some requirements for the repos functionality. Closes #1075

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]

File: charts/otel-cloud-stack/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: otel-cloud-stack
```
Comment (TylerHelmuth) [inline charts/otel-cloud-stack/Chart.yaml:2]: Whats the origin for this name.  If the goal is to be like `kube-prometheus-stack` should we use `kube-otel-stack`? Or to stick with our existing naming convention, should we do `opentelemetry-kube-stack`?

**Review by TylerHelmuth — COMMENTED** [review]
@open-telemetry/helm-approvers please take a look.

File: charts/otel-cloud-stack/Chart.yaml (hunk 1 above)
Comment (jaronoff97) [reply charts/otel-cloud-stack/Chart.yaml:2]: @TylerHelmuth im fine with anything to be honest. I originally made kube-otel-stack, and then i created a new chart that was entirely otel-based and wanted something differentiating so i did otel-cloud-stack. I'm fine with whatever you all think is best.

Comment (TylerHelmuth) [reply charts/otel-cloud-stack/Chart.yaml:2]: I vote goes to `opentelemetry-kube-stack` to match our existing naming convention.

Comment (Toaddyan) [reply charts/otel-cloud-stack/Chart.yaml:2]: Kube otel stack is more synonymous to kube prom stack. ( it also does much more than just metrics) But following naming conventions is more important.

Comment (Allex1) [reply charts/otel-cloud-stack/Chart.yaml:2]: kube-otel-stack or otel-kube-stack

Comment (dmitryax) [reply charts/otel-cloud-stack/Chart.yaml:2]: `opentelemetry-kube-stack` sounds good to me

Comment (TylerHelmuth) [reply charts/otel-cloud-stack/Chart.yaml:2]: @povilasv @puckpuck @JaredTan95 please share your opinion.

Comment (jaronoff97) [reply charts/otel-cloud-stack/Chart.yaml:2]: that sounds good to me... changing now.

File: .github/workflows/kube-stack-test.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Test Otel Cloud Stack Charts
```
[the remark this answers is not in the document]
Comment (jaronoff97) [reply .github/workflows/kube-stack-test.yaml:1]: this is what happens when you use sed 😮‍💨

**Review by TylerHelmuth — APPROVED** [review]

**Review by Allex1 — APPROVED** [review]

**Pull request merged**